main.c

```diff
@@ -137,9 +138,31 @@ void hexdump(const void *memory, size_t length)
 }
 #else
 static void print(const char * unused) { }
+static void print_p(const void * unused) { }
```

Comment:
Stale debugging?

Reply:
No, it is used to print pointer and integer values